src/Minter.sol

```diff
@@ -679,6 +680,14 @@ contract Minter is IMinter, AccessControlDefaultAdminRules, ReentrancyGuard, Lin
      * @param amount Transfer amount.
      */
     function saveFunds(address paymentToken, address to, uint256 amount) public onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Also we should have a test for this function

Reply:
fixed